src/NzbDrone.Core/ImportLists/PTP/PTPSettings.cs

```diff
@@ -0,0 +1,44 @@
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.PTP;
+
+public class PTPSettingsValidator : AbstractValidator<PTPSettings>
+{
+    public PTPSettingsValidator()
+    {
+        RuleFor(c => c.BaseUrl).ValidRootUrl();
+        RuleFor(c => c.APIUser).NotEmpty();
+        RuleFor(c => c.APIKey).NotEmpty();
+        RuleFor(c => c.ID).GreaterThan(0);
+    }
+}
+
+public class PTPSettings : IProviderConfig
+{
+    private static readonly PTPSettingsValidator Validator = new PTPSettingsValidator();
+
+    public PTPSettings()
+    {
+        BaseUrl = "https://passthepopcorn.me";
+    }
+
+    [FieldDefinition(0, Label = "URL", Advanced = true, HelpText = "Don't change this unless you know what you're doing, since your API key will be sent to this host.")]
```

Comment:
should this even be a setting, or should it just be hardcoded?

Reply:
This is how every other natively-supported indexer has it, I was just following their lead. I personally think it should be removed from every natively-supported indexer, since the setting shouldn't be need to used; things like TLS specific subdomains are a thing of the past.

Reply:
Agree that these should all be removed in the future, but that's for another PR. I'm good either way for this one.